package.json

```diff
@@ -41,6 +41,8 @@
     "react-redux": "^4.4.1",
     "react-router": "^2.4.1",
     "react-router-redux": "^4.0.2",
+    "react-select": "^1.0.0-rc.5",
+    "react-select-plus": "^1.0.0-rc.5",
```

Comment:
I'm using `react-select-plus`.  I thought I'd removed `react-select`, but apparently not - I'll fix that.  Thanks!